src/dropdown.ts

```diff
@@ -861,6 +861,7 @@ export default class GlideCoreDropdown
                   ? '-1'
                   : '0'}
                 type="button"
+                ?disabled=${this.disabled}
```

Comment:
Fixes [this](https://github.com/CrowdStrike/glide-core/pull/751#discussion_r1988148934) and [also](https://glide-core.crowdstrike-ux.workers.dev/add-aria-snapshot-testing/visual-test-report#?testId=3b43c9d186bbf2d5163d-784d341e915cd4cdde01) happened to fix a visual bug.